src/app/user/login/login.component.scss

```diff
@@ -1,5 +1,81 @@
-.login-main-container {
-    height: 90vh;
-    background-color: red;
-    flex: 1;
-}
\ No newline at end of file
+.main-container-login {
```

Comment:
you can do a lot of the size setting changes with pure flex and it may make the styling more maintainable in the future; however, don't need to revert these changes and you can just resolve this comment without making any changes if you don't want to, as it doesn't affect usage (rem should update proportionally based on window size just like flex I believe)